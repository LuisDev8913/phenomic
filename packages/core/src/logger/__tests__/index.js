import logger from "..";

test("log should log", () => {
  const log = logger("sender test", false);
  expect(log("test msg")).toMatchSnapshot();
  expect(log.debug("test debug")).toMatchSnapshot();
  expect(log.info("test info")).toMatchSnapshot();
  expect(log.success("test success")).toMatchSnapshot();
  expect(log.warn("test warning")).toMatchSnapshot();
  expect(log.error("test error")).toMatchSnapshot();
});
