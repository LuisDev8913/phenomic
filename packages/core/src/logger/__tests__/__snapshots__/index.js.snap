// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`log should log 1`] = `"[34mâ„¹[39m [90msender test: [39m test msg"`;

exports[`log should log 2`] = `"  [90msender test: [39m test debug"`;

exports[`log should log 3`] = `"[34mâ„¹[39m [90msender test: [39m test info"`;

exports[`log should log 4`] = `"[32mâœ”[39m [90msender test: [39m test success"`;

exports[`log should log 5`] = `"[33mâš [39m [90msender test: [39m test warning"`;

exports[`log should log 6`] = `"[31mâœ–[39m [90msender test: [39m test error"`;
